src/conditional-flow/string-conditions.js

```diff
@@ -47,16 +77,33 @@ function getMiddleLetter (val1) {
 // Summer - June to August
 // Autumn - September to November
 // Winter - December to February
-function seasonForMonth (monthName) {
-
+function seasonForMonth(monthName) {
   // TODO: write code in this function body to pass the tests
+  Spring= ['March','April','May']
+  Summer= ['June','July','August']
```

Comment:
Great use of arrays. Just be aware that you should create your variables like so:
```js
const Spring = ['Match', 'April', 'May']
```